### ISSUE TITLE
Regarding #545

### ISSUE THREAD

**Issue by thad0ctor** [jettify/pytorch-optimizer#546]
I would have posted inline in #545 but @ndvbd blocked my account after posting evidence in a now-deleted issue on [the Gefen repo](https://github.com/ndvbd/Gefen) showing that his paper claims are incorrect: 

Before the maintainers consider this as feature I would like to point out that independent verification of their paper claims and repo claims do not check out. 

Normally I would chalk this up to someone being ambitious and trying to push the boundary of SOTA (or even cherry picking benchmarks) but after I filed a PR (# 2), the maintainer put most of PR # 2's changes onto main as a commit they authored themselves rather than merging the PR... this coincided with the deletion of issue # 1 where I and another user were exchanging our findings which undermined the repo's claims and subsequently blocking me.  I can't speak to motives but I find this all very sus so I'm going to call them out before anyone else spends time validating this optimizer. 

[Issue #1 ](https://github.com/ndvbd/Gefen/issues/1#issuecomment-4802448972) (now deleted) highlighted:

- broken FSDP (claimed working in repo/paper) - https://github.com/ndvbd/Gefen/pull/3
- non-functional checkpoint resumption functionality - https://github.com/ndvbd/Gefen/pull/2
- missing paths for muon 1D matrix fallback (despite being touted as a "drop-in" replacement) - obvious in code
- inability by to reproduce VRAM savings and throughput increase claims - see logs

Since then I have discovered: 
- learning rate parity claims  vs. adamw do not check out on modern models (e.g. qwen 3)-  see logs

I will admit the underlying idea has potential but the very behavior of this team should call a lot into question, regardless, this is nowhere near the level of maturity this feature request would imply especially given the seeming legitimacy of their paper/repo claims and the PRs they have opened across major training platforms.

Axotol logs below so you don't just take my word for it.

[gefen.log](https://github.com/user-attachments/files/29399503/gefen.log)

[adamw.log](https://github.com/user-attachments/files/29399505/adamw.log)
